includes/admin/OrderSettings.php

```diff
@@ -246,6 +259,70 @@ private function setAllData(): void
         $this->setDigitalStampRangeWeight();
     }
 
+    // TODO refactor this (MY-28781)
+    public function setShippingRecipient(): self
+    {
+        $thisOrder  = $this->order;
```

Comment:
doe maar gewoon $order, dit geeft me flashbacks naar javascript frameworks waar ze "spelen" met `this` 😛